.github/workflows/cmake_build.yml

```diff
@@ -112,7 +112,8 @@ jobs:
               -B build \
               -G "${{matrix.config.generator}}" \
               -A ${{matrix.config.platform}} \
-              -D audacity_use_pch=no
+              -D audacity_use_pch=no \
+              -D audacity_has_networking=yes 
```

Comment:
Changes in this file turn networking ON for CI builds, but that won't apply by default to anyone's DIY builds.